_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -36,7 +36,7 @@ knitr_fig_path("01-starting-with-data-")
 ```
 
 We are studying inflammation in patients who have been given a new treatment for 
-arthritis, and need to analyze the first dozen data sets.
+arthritis, and need to analyze the first dozen data sets ([download data](http://swcarpentry.github.io/r-novice-inflammation/data/r-novice-inflammation-data.zip)).
```

Comment:
```suggestion
arthritis, and need to analyze the first dozen data sets (here is a link to [download the data for this lesson]({{ page.root }}/data/r-novice-inflammation-data.zip)).
```

Thank you! I've changed the URL to avoid link rot and text for a more precise description